src/ResourceManager/Websites/Commands.Websites/help/Set-AzureRmWebApp.md

```diff
@@ -77,6 +77,34 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### -AsJob
+Run cmdlet in the background
+
+```yaml
+Type: SwitchParameter
+Parameter Sets: (All)
+Aliases: 
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -AssignIdentity
+Enable MSI on an existing azure webapp or functionapp```yaml
```

Comment:
@panchagnula there trailing ` ```yaml ` string on this line should be moved to a line by itself (see the above `AsJob` parameter as an example). This is a bug in the `platyPS` module. Please do the same for the `HttpsOnly` parameter below.

Reply:
Got it - sorry didn't notice this earlier.